ssl/ssl_lib.c

```diff
@@ -632,8 +632,9 @@ SSL *SSL_new(SSL_CTX *ctx)
      * chained DRBG.
      */
     if (RAND_get_rand_method() == RAND_OpenSSL()) {
-        s->drbg = RAND_DRBG_new(NID_aes_128_ctr, RAND_DRBG_FLAG_CTR_USE_DF,
-                                RAND_DRBG_get0_global());
+        s->drbg =
+            RAND_DRBG_new(RAND_DRBG_NID, RAND_DRBG_FLAG_CTR_USE_DF,
+                          RAND_DRBG_get0_global());
         if (s->drbg == NULL
             || RAND_DRBG_instantiate(s->drbg, NULL, 0) == 0) {
```

Comment:
I guess we don't need a personalization string here since the input is coming from a "personalized" instance of the DRBG?

Reply:
I am going to use the existing SSL_version_str as personalization string.